exporters/stdout/stdoutlog/exporter_test.go

```diff
@@ -219,7 +219,7 @@ func getPrettyJSON(now time.Time) string {
 	"TraceID": "0102030405060708090a0b0c0d0e0f10",
 	"SpanID": "0102030405060708",
 	"TraceFlags": "01",
-	"Resource": {},
+	"Resource": null,
```

Comment:
Can we add a test that includes a serialized resource?

Reply:
IIUC, there is no way to set the resource outside the `sdk/log` package.